### DISCUSSION TITLE
[Feature Request] Backup Encryption

### DISCUSSION THREAD

**Discussion by vzool** [helloworld1/FreeOTPPlus#247] [Ideas]
Backup Encryption is essential for safety and security 🤗

**Comment by fizzlifax2** [comment]
I would like to have any encrypted file or even the option to integrate this in keepass or similar passwordmanager!
To export an ascii file with the tokens on my smartphone seems me really too risky!
Since I dont like Google at all - I would like to put my token- and password-file especially to my Dropbox shares...!??
Is this still possible? - And how does it work?
Thanks a lot in adavance!
Fi
